### ISSUE TITLE
Scalars in response body not correctly supported.

### ISSUE THREAD

**Issue by greemo** [wundergraph/wundergraph#806]
### Bug description

When returning a scalar from a graphql backend, it is expecting a string, although the schema states scalar.

### How to reproduce

A Wundergraph server configured to reproduce the error: 
[wgtest.zip](https://github.com/wundergraph/wundergraph/files/11168836/wgtest.zip)

A simple graphql server which acts as a backend (I could not reproduce with the built-in graph-ql server unfortunately): 
[graphql-test.zip](https://github.com/wundergraph/wundergraph/files/11168841/graphql-test.zip)

Run the graphql server with `yarn && node server.js`

Run the Wundergraph server with 'yarn && npx wunderctl up --debug`

Open the Wundergraph graphql server at: http://127.0.0.1:9991/graphql and send the following request:

`
query {
  b1_bar(foo: 222) {
    foo
  }
}
`

You will receive as response:
 
`
{
  "errors": [
    {
      "message": "could not resolve response"
    }
  ]
}
` 

And something similar to this is the Wundergraph console:

`2023-04-06T14:34:01+10:00 ERROR apihandler/apihandler.go:888 ResolveGraphQLResponse {"component": "@wundergraph/node", "reqId": "8fbe8a40-a609-ef52-bb7a-1fd94f8132a0", "error": "invalid value type 'number' for path /data/b1_bar/foo, expecting string, got: 123. You can fix this by configuring this field as Int/Float/JSON Scalar"}`

### Expected behavior

The scalar is correctly returned from the GraphQL backend to the client making the request to Wundergraph.

### WunderGraph information

See attached example program.

### Environment & setup

Linux Mint 20
Go 1.13.8
Node 18.12.1

### WunderCtl Version

See attached example program.

**Comment by jensneuse** [comment]
What is the encoding of the custom scalar?

**Comment by greemo** [comment]
Hi @jensneuse, thanks!

When I do that, the type is removed, and I need to define any operations using Int, 

For example, in the project I provided, the following operation was defined:
`
query b1GetIt($id: b1_BigInty!) {
	b1_bar(foo: $id) {
		foo
	}
}
`

However after adding the customIntScalars configuration, my operation becomes as follows because the scalar is no longer available:
`
query b1GetIt($id: Int!) {
	b1_bar(foo: $id) {
		foo
	}
}
`

The values being passed via the BigInty type are larger than a 32 bit number, so this solution seems inappropriate.

I've just noticed that if I define the scalar in customJSONScalars, the typename is used correctly and everyhing works.

I'm happy to close this now, as it seems I can do what I need to defining the scalar as a customJSONScalar. This makes sense, as by default, scalars in GraphQL must be able to be represented in JSON.

Thinking more about this, It may make sense to suggest users to add all their graphql scalars as customJSONScalars (with a warning during the generate phase) or even automatically add them if not added explicitly to customIntScalars or customStringScalars for example.
